src/utils/u60_representation.nr

```diff
@@ -94,6 +118,45 @@ impl<let N: u32, let NumSegments: u32> U60Repr<N, NumSegments> {
         result
     }
 
+    pub(crate) unconstrained fn from_field(input: Field) -> Self {
+        let (first, second, third, fourth, fifth) = unsafe { field_to_u60rep(input) };
```

Comment:
This should be complaining about unnecessary unsafe blocks now. @kashbrti do you have the noir vscode extension installed?